src/app/dataset-flow/dataset-flow-details/tabs/flow-details-history-tab/flow-details-history-tab.helpers.ts

```diff
@@ -136,10 +137,17 @@ export class DatasetFlowDetailsHelpers {
                                                       ? "block"
                                                       : "blocks"
                                               }`
-                                            : "Dataset is up-to-date";
+                                            : flowDetails.description.ingestResult?.__typename ===
+                                                    "FlowDescriptionUpdateResultUpToDate" &&
+                                                flowDetails.description.ingestResult.uncacheable &&
+                                                flowDetails.configSnapshot?.__typename === "FlowConfigurationIngest" &&
+                                                !flowDetails.configSnapshot.fetchUncacheable
+                                              ? "Source is uncacheable, use force update to re-scan the data"
```

Comment:
Why the template here different than above?

Reply:
Fixed to message as for the main table